config/lighthouse.php

```diff
@@ -0,0 +1,517 @@
+         */
+        'enable' => env('LIGHTHOUSE_CACHE_ENABLE', 'local' !== env('APP_ENV')),
+
+        /*
+         * Allowed values:
+         * - 1: uses the store, key and ttl config values to store the schema as a string in the given cache store.
+         * - 2: uses the path config value to store the schema in a PHP file allowing OPcache to pick it up.
+         */
+        'version' => env('LIGHTHOUSE_CACHE_VERSION', 1),
+
+        /*
+         * Allows using a specific cache store, uses the app's default if set to null.
+         * Only relevant if version is set to 1.
+         */
+        'store' => env('LIGHTHOUSE_CACHE_STORE', null),
+
+        /*
+         * The name of the cache item for the schema cache.
+         * Only relevant if version is set to 1.
+         */
+        'key' => env('LIGHTHOUSE_CACHE_KEY', 'lighthouse-schema'),
+
+        /*
+         * Duration in seconds the schema should remain cached, null means forever.
+         * Only relevant if version is set to 1.
+         */
+        'ttl' => env('LIGHTHOUSE_CACHE_TTL', null),
+
+        /*
+         * File path to store the lighthouse schema.
+         * Only relevant if version is set to 2.
+         */
+        'path' => env('LIGHTHOUSE_CACHE_PATH', base_path('bootstrap/cache/lighthouse-schema.php')),
+
+        /*
+         * Should the `@cache` directive use a tagged cache?
+         */
+        'tags' => false,
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Query Cache
+    |--------------------------------------------------------------------------
+    |
+    | Caches the result of parsing incoming query strings to boost performance on subsequent requests.
+    |
+    */
+
+    'query_cache' => [
+        /*
+         * Setting to true enables query caching.
+         */
+        'enable' => env('LIGHTHOUSE_QUERY_CACHE_ENABLE', true),
+
+        /*
+         * Allows using a specific cache store, uses the app's default if set to null.
+         */
+        'store' => env('LIGHTHOUSE_QUERY_CACHE_STORE', null),
+
+        /*
+         * Duration in seconds (minutes for Laravel pre-5.8) the query should remain cached, null means forever.
+         */
+        'ttl' => env(
+            'LIGHTHOUSE_QUERY_CACHE_TTL',
+            \Nuwave\Lighthouse\Support\AppVersion::atLeast(5.8)
+                ? 24 * 60 * 60 // 1 day in seconds
+                : 24 * 60 // 1 day in minutes
+        ),
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Namespaces
+    |--------------------------------------------------------------------------
+    |
+    | These are the default namespaces where Lighthouse looks for classes to
+    | extend functionality of the schema. You may pass in either a string
+    | or an array, they are tried in order and the first match is used.
+    |
+    */
+
+    'namespaces' => [
+        'models' => ['App', 'App\\Models'],
```

Comment:
Why not put it on Graphql namespace? @kaioken

Reply:
done